ci/jenkins/test.sh

```diff
@@ -530,7 +530,7 @@ function run_e2e {
     if [[ $FLEXIBLE_IPAM == true ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
     else
-        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus
+        go test -run=TestMulticast/testMulticaststats -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=20m --prometheus
```

Comment:
this  should be removed before merging ?

Reply:
updated.